includes/definitions/technicolor.yaml

```diff
@@ -0,0 +1,13 @@
+os: technicolor
```

Comment:
The os name should be called after the firmware the device runs like IOS for Cisco IOS. Using the vendor name isn't really the best option, what can be used instead?

Reply:
I don't find many information about it.
I've only this:
_COMPANY_NAME=Technicolor
_COMPANY_URL=http://www.technicolor.com
_PROD_NAME=Technicolor TG
_BRAND_NAME=MediaAccess


Maybe we can call it "tg-os" or "mediaaccess-os".

Reply:
tgos would be ok if the TG devices run the same firmware